applications/common/backend/src/main/java/pl/ds/kyanite/common/components/models/table/TableHeadCellComponent.java

```diff
@@ -16,39 +16,11 @@
 
 package pl.ds.kyanite.common.components.models.table;
 
-import javax.annotation.PostConstruct;
-import javax.inject.Inject;
-import lombok.Getter;
 import org.apache.sling.api.resource.Resource;
-import org.apache.sling.models.annotations.Default;
 import org.apache.sling.models.annotations.DefaultInjectionStrategy;
 import org.apache.sling.models.annotations.Model;
 
 @Model(adaptables = Resource.class, defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
-public class TableHeadCellComponent {
```

Comment:
The code was identical as in `TableCellComponent`.

Refactor to extend from that class, so all changes are inherited